### DISCUSSION TITLE
Sync phase - one per application or one per Git repo?

### DISCUSSION THREAD

**Discussion by peterwojcicki** [argoproj/argo-cd#6699] [Q&A]
Our setup involves a single github repository (which Argo watches) which contains multiple Argo applications. 
When Argo performs the sync operation - does it perform one Sync phase for the entire git repository, or does it perform multiple sync operations - one Sync per each of the Argo applications?

**Comment by cnukwas** [comment]
I am still learning, but it's my understanding that sync happens at the folder you give during Application creation, not at the entire git repo. There are options for Directory recursive or something similar, if your YAML or Helm templates are in different directories. So it will be a sync per Application unless my understanding is not right.
